data_scraper/tests.py

```diff
@@ -1,3 +1,36 @@
+import os
+
 from django.test import TestCase
+from selenium import webdriver
+from selenium.webdriver.support.wait import WebDriverWait
+
+import data_scraper.scra as scra
+
+
+class FindTestCase(TestCase):
+    def setUp(self):
+        # Animal.objects.create(name="lion", sound="roar")
+        pass
 
-# Create your tests here.
+    def test_find_dados_via_webdriver(self):
```

Comment:
@salarini-e add a test with `find_dados_via_bs`. should be almost the same code.